### DISCUSSION TITLE
Contribute to project in future.

### DISCUSSION THREAD

**Discussion by toomsie** [ramda/ramda#3251] [General]
Hello my name is Gerald. Would like to contributing to the project soon, as soon as I get up to speed understanding Ramda and polish up my understanding of functional programming as it has been a long time since I last used SmallTalk. This may take a few weeks at best. 
Kind regards
Gerald

**Comment by CrossEye** [comment]
@toomsie:

Welcome aboard Gerald!  We are trying to make our way soon to a long-overdue version `1.0`.  There's plenty to do for that, and after that, there's lots of work to do to modernize Ramda.  If you have questions about how things are working, feel free to ask questions here.  Also feel free to email me directly at scott@sauyet.com, but here (or on StackOverflow) you'll have more eyes on your questions.

BTW, SmallTalk is not at all a functional language.  It's Object-oriented, to some the epitome of OO.

**Reply by toomsie** [reply]
Thanks very much Scott well appreciated.
